thirdweb/deploy.yaml

```diff
@@ -0,0 +1,31 @@
+version: "2.0"
+services:
+  web:
+    image: "thirdweb/starter-kits:latest"
```

Comment:
Please change this tag to a specific version. Akash aggressively caches them so users will get a different ":latest" tag.

Reply:
Hey Andy, checked our docker image over here: https://hub.docker.com/r/thirdweb/starter-kits/tags

We realized we only have one tag "latest" do we still have to specify a version if we only have one tag available?

![image](https://user-images.githubusercontent.com/85150796/196024481-2ce042b3-8de0-429b-94ae-c98da2ea674d.png)